### DISCUSSION TITLE
Rowy no longer has access to your Firebase project

### DISCUSSION THREAD

**Discussion by darkmayuk** [AntlerVC/firetable#1490] [Support / Q&A] [answered]
I have always managed my Firebase project through Rowy just fine up to now. But on logging in to Rowy today I was greeted by the message "Rowy no longer has access to your Firebase project [project name]". I haven't changed anything on the Firebase side or the Rowy side, at least not intentionally. Any ideas how to solve this? Thanks.

**Comment by Chadha93** [comment, marked as answer]
Hi @darkmayuk, there were a couple of recent updates made in Rowy with the v3.0.0 release and after it. You can reconnect your Rowy project again. The team has enhanced the project setup process in Rowy. 

Feel free to post your question in Discord on the #rowy-help channel, if you encounter any issues while connecting again. 
Discord - https://www.rowy.io/discord

Regards,
Gaurav

**Comment by darkmayuk** [comment]
Hi there - so, to be clear, are you saying that I need to connect
(re-connect) the Rowy project to the Firebase db? If so, can you point me
to the most up to date set of instructions to follow? Further, do I create
a whole new 'Workspace' in Rowy, or do I somehow fix the existing Workspace
that I have there?

Thanks,
Mark

**Comment by darkmayuk** [comment]
Hello - I am reaching out one last time to see if you will help me. I have
sent my question to you - I have posted it on the github forum - I have
posted it on the Discord server. No one has replied! Please, please help -
my Rowy connection is now unusable following your update. And your
documentation does not cover this case.

My question was: [following error " Access Denied. Rowy no longer has
access to your Firebase project singzing-f3a31." and your comment " You can
reconnect your Rowy project again "] ... so, to be clear,

*are you saying that I need to connect (re-connect) the Rowy project to the
Firebase db? *

*If so, how do I do this? Can you point me to the most up to date set of
instructions to follow? *

*Further, do I create a whole new 'Workspace' in Rowy, or do I somehow fix
the existing Workspace that I have there?*

Please, answer my questions, otherwise I will have to abandon Rowy and try
to find a different tool to manage my Firebase.

Thanks.

Mark Day

**Reply by Chadha93** [reply]
@darkmayuk Hello, There is already a followup question here on your Discord - https://discord.com/channels/853498675484819476/1179052855131377704. 

Also, we have seen a couple of increase in similar requests, we are working on fixing all the project setup issues on high priority. 

Could you please share a quick video of the access denied issue here or on the above discord thread link? cc @bofeiw

**Comment by darkmayuk** [comment]
Hi Gaurav, I've replied to you on Discord, with a screenshot there.

I really hope you can fix this very soon - at the moment I have no access
to my Firebase at all.

Thanks.

Mark

**Comment by bofeiw** [comment]
Hi @darkmayuk, sorry for the late reply. Is this still an issue? 

You can try reconnecting the project:
1. In any workspace, click create project
2. Sign in to Google and select the project to reset
3. You will see the "disconnect" button, use it to disconnect it from Rowy. Your Firebase data will be untouched.
4. After disconnecting it, you can setup the project again in the same page.

<img width="872" alt="Screenshot 2023-12-21 at 20 58 02" src="https://github.com/rowyio/rowy/assets/34177142/4679d44e-1374-42f1-a70d-6395e0b026fc">

Please let us know if you if it works.

**Comment by darkmayuk** [comment]
Hi - thanks for getting back to me. Unfortunately your instructions about
exactly how to disconnect and reconnect were too late - they came over a
month after the connection failed - I had to take action before then! So I
had already deleted the project in Rowy, then recreated it. You can see
more details in my messages on Discord. The result is that it now works,
sort-of. Rowy is connected to Firebase, and one table/ collection works,
but the other one does not.

As I just said on Discord:* I only have two tables in the project. One
'videos' is working fine. The other 'algorithm' is not working. It is
listed in my Rowy workspace. When I click on it Rowy points out that the
collection exists, so I should import data. Then I try to import data, and
it throws this error.*

I seem to get a different error ID each time I try this, but the latest one
is ROWY TRACING ID: ROWYERR-3B40-C7F7-5802-B164ECC219B2

Please let me know how to proceed.

Thanks,

Mark

**Reply by bofeiw** [reply]
Continuing the table import issue in Discord [here](https://discord.com/channels/853498675484819476/1179052855131377704/1187699502728101910), closing this thread as the original issue is resolved.
